release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1191,6 +1195,10 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-image-registry-bug-fixes"]
 ==== Image Registry
 
+* Previously, the Image Registry Operator is degraded. This can happen if the pruner fails. The operator remains degraded until the pruner successfully runs. By default, the pruner runs once a day, but can be run more often. When a successful run is completed, the Image Registry Operator is no longer degraded. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1990125[*BZ#1990125*])
```

Comment:
This seems more like a known issue than a bug fix:

```suggestion
* If the pruner fails, the Image Registry Operator is degraded until the pruner successfully runs. By default, the pruner runs once a day. As a workaround, you can give it more opportunities to run successfully by configuring it to run more often. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1990125[*BZ#1990125*])
```

Ideally would want to tell them _how_ to make the pruner run more frequently.

Reply:
Thats what I was thinking too.... It felt kinda weird to put it in the bug fix section.